### ISSUE TITLE
"Skip to main content" works inconsistently

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#2505]
AT users are having difficulty using the skip to main content. Some users try to click the link and are taken nowhere; others confuse the landmark with the link and click it twice. Tests conducted in Edge with JAWS

[Fable video clip 1](https://app.makeitfable.com/share/64d3bacdf2b22e001341e360)
[Fable video clip 2](https://app.makeitfable.com/share/64d3bd3136ac750018651b4e)
[Fable video clip 3](https://app.makeitfable.com/share/64d3be56f2b22e001341e361)
[Fable video clip 4](https://app.makeitfable.com/share/64d3bfc8f2b22e001341e362)
[Fable video clip 5](https://app.makeitfable.com/share/64d3c2c0f2b22e001341e364)

**Comment by dsamojlenko** [comment]
Having reviewed all of these videos, I am unable to recreate the issues described. Possibly because of different tooling, possibly because the issues may be intermittent, unsure. 

However, in the second video link, the participant indicates that having a nav landmark region around a single "Skip to main content" link is probably unnecessary. [This WebAIM article](https://webaim.org/techniques/semanticstructure/#regions) supports that assessment.

**Comment by connorscarolyns** [comment]
@thiessenp-cds  You can't see the clips?

**Comment by thiessenp-cds** [comment]
> @thiessenp-cds You can't see the clips?

Op now I have access. I just need to be more patient I guess :)

**Comment by thiessenp-cds** [comment]
I wasn't able to get the same behaviour either. It may just be a one off quirk. IMO we can note this and move on. Though if another user gets similar behaviour then maybe come back to it.

Though to be really sure, I'd need to know what version of Jaws the user is running. Each screen reader has its own quirks/bugs :) From what I can see he's running windows 10, Edge latests, with Jaws. I tested on Windows 10, Edge latest and Jaws  v2023.2306.38

**Comment by thiessenp-cds** [comment]
The other issue he mentions about where the Skip link takes a user, I guess that is up for discussion. For my opinion, I agree with what the user is saying - that a skip to main content link would ideally focus the main content heading. This just makes the interaction clearer. (vs. focusing the main element region)

**Comment by connorscarolyns** [comment]
@Abi-Nada

**Comment by thiessenp-cds** [comment]
The work for this will involve updating each skip to link to point to each page header. Except for forms-forms, I think I'd like to do that in a separate PR because of how impactful that change could be.

**Comment by thiessenp-cds** [comment]
I found a simple way to solve this on the below branch. I'll put this task in the backlog for now so we can focus on the download task

https://github.com/cds-snc/platform-forms-client/pull/2813

**Comment by thiessenp-cds** [comment]
Completed in #5883
